godel_surface_detection/src/services/surface_blending_service.cpp

```diff
@@ -247,11 +247,20 @@ void SurfaceBlendingService::save_blend_parameters(const std::string& filename)
 
 bool SurfaceBlendingService::load_path_planning_parameters(const std::string & filename)
 {
+  using godel_param_helpers::loadParam;
+
   if(godel_param_helpers::fromFile(filename, path_planning_params_))
   {
     return true;
   }
-  return false;
+  // otherwise default to the parameter server
+  ros::NodeHandle nh ("path_planning_params");
```

Comment:
Other parameter _sets_ are read from specific namespaces (e.g. [`"~/blend_plan"`](https://github.com/tecnalia-advancedmanufacturing-robotics/godel/blob/94a63a2ad693f3d7941fe35584b481da4080ae47/godel_surface_detection/src/services/surface_blending_service.cpp#L225)). Shouldn't this be done similarly?